pkg/apis/core/validation/validation.go

```diff
@@ -3397,7 +3397,11 @@ func ValidatePodSecurityContext(securityContext *core.PodSecurityContext, spec *
 		}
 
 		if securityContext.Sysctls != nil {
```

Comment:
I guess the decoder will give us non-nil on an empty list? Do we want to reject an empty list?

Reply:
nil or empty list has the same semantics here. I will update it to check the len.